File: py110/small_problems_110_119/medium2/4.py
# https://launchschool.com/exercises/2fb67370

"""


"""
import datetime

def friday_the_13ths(year):

    thirteen = [datetime.date(year, month, 13) for month in range(1, 13)]
    
    num_of_fridays = 0

    for days in thirteen:
        if datetime.datetime.isoweekday(days) == 5:
            num_of_fridays += 1
    
    return num_of_fridays
    



print(friday_the_13ths(1986) == 1)      # True
print(friday_the_13ths(2015) == 3)      # True
print(friday_the_13ths(2017) == 2)      # True

File: ls215/smallProblems/med2/4.js
// https://launchschool.com/exercises/28e1b056

'use strict';

function fridayThe13ths(year) {
  let array = [];

  for (let i = 0; i < 12; i += 1) {
    let date = new Date(year, i, 13);
    array.push(date);
  }

  return array.filter(date => date.getDay() === 5).length;

}



console.log(fridayThe13ths(1986));      // 1
console.log(fridayThe13ths(2015));      // 3
console.log(fridayThe13ths(2017));      // 2